src/code42cli/cmds/devices.py

```diff
@@ -611,6 +612,7 @@ def handle_row(**row):
             row["reactivated"] = "True"
         except Exception as err:
             row["reactivated"] = f"False: {err}"
+            raise
```

Comment:
Will this change (and similar in the other handle_row methods) cause the output at the end to not include the failures?

Reply:
It does include failures now _with_ this change. Before, it was all successes.
This error is again caught somewhere that manages that stats (worker.py)

Reply:
If no error is raised in the handler, the stats incrementation logic considers the row a success

Reply:
hmm...  I guess that individual status of the rows failures no longer show though...
ugh.

Reply:
Yeah, it was the output of all rows at the end that concerned me. Ideally it would be nice if we could both show the failures in the status as the job runs and also output them all at the end, but I'm not sure how best to actually accomplish that with the way it's structured currently.

Reply:
I have an idea to make both worlds work. BRB!

Reply:
Ok now it should work with both -- the bulk commands that need to display the row status as it goes along can do that and still have their final results correct at the end.